components/core/src/clp/ffi/ir_stream/search/test/test_QueryHandlerImpl.cpp

```diff
@@ -249,10 +251,14 @@ auto get_matchable_values(SchemaTree::Node::Type node_type) -> std::vector<Value
             matchable_values.emplace_back(fmt::format("ThisIs{}", cRefTestStr));
             auto const long_str{fmt::format("This is {}", cRefTestStr)};
             matchable_values.emplace_back(
-                    get_encoded_text_ast<ir::four_byte_encoded_variable_t>(long_str)
+                    EncodedTextAst<ir::four_byte_encoded_variable_t>::parse_and_encode_from(
```

Comment:
```suggestion
                    FourByteEncodedTextAst::parse_and_encode_from(
```
Personal taste, but I think its easier to read without templates. Leaving suggestions for the other instances in this file if you agree.